src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write your code here
+// head and foot
+const thead = document.querySelector('thead');
+const tfoot = document.querySelector('tfoot');
+
+[thead, tfoot].forEach((el) => {
+  const lastElement = el.querySelector('tr').lastElementChild;
+  const title = document.createElement('th');
+
+  title.innerText = 'Position';
+
+  lastElement.insertAdjacentElement('beforebegin', title);
+});
+
+// body
+const tbody = document.querySelector('tbody');
+
+const people = [...tbody.children];
+
+people.forEach((person) => {
+  const position = [...person.children][1].cloneNode(true);
```

Comment:
The code assumes that each 'person' (table row) has at least two children. If a row has fewer than two cells, this will cause an error. Consider adding a check to ensure there are enough cells before attempting to clone the second one.